.gitignore

```diff
@@ -6,3 +6,4 @@ notes.md
 tests/fixtures/cassette*
 /.php_cs.cache
 /.php_cs
+composer.lock
```

Comment:
I don't remember why composer.lock is tracked, but I would suggest that it's removal is done in a separate PR.